src/navigator/types.tsx

```diff
@@ -35,8 +35,8 @@ import {
 // Typed nested navigator params
 type NestedNavigatorParams<ParamList> = {
   [K in keyof ParamList]: undefined extends ParamList[K]
-    ? { screen: K; params?: ParamList[K] }
-    : { screen: K; params: ParamList[K] }
+    ? { screen: K; params?: ParamList[K]; showBackButton?: boolean }
+    : { screen: K; params: ParamList[K]; showBackButton?: boolean }
```

Comment:
I guess this is only used in the QRNavigator, but this doesn't seem the right place to add this prop since NestedNavigatorParams sounds like it could be used with other screens. Can we make this QR screen specific?